data/processors/maps.py

```diff
@@ -126,6 +126,23 @@ def assign_coordinates(data):
         raise RuntimeError("Aborting due to errors")
 
 
+def check_coords(input_data):
+    """ Check for issues with coordinates """
+
+    for iid, data in input_data.items():
+        if data["type"] == "root":
+            continue
+
+        if data["coords"]["lat"] == 0. or data["coords"]["lon"] == 0.:
+            print(f"{iid}: lat and/or lon coordinate is zero. If unknown, omit this field.")
+            continue
```

Comment:
```suggestion
            print(f"{iid}: lat and/or lon coordinate is zero. Please provide an accurate coordinate!")
            exit(1)
```
I think it would be better to fail at this point, because we shouldn't have wrong data compiled in (I know this is old code from me)